TAO/orbsvcs/orbsvcs/SSLIOP/SSLIOP_Factory.cpp

```diff
@@ -411,6 +414,17 @@ TAO::SSLIOP::Protocol_Factory::init (int argc, ACE_TCHAR* argv[])
             }
         }
 
+      else if (ACE_OS::strcasecmp (argv[curarg],
+                                   ACE_TEXT("-SSLCRLFile")) == 0)
+        {
+          curarg++;
+          if (curarg < argc)
+            {
+              crl_type = parse_x509_file (ACE_TEXT_ALWAYS_CHAR(argv[curarg]),
```

Comment:
Add new option to TAO/docs/Security/SSLIOP-USAGE.html

Reply:
> Add new option to TAO/docs/Security/SSLIOP-USAGE.html

done